.azure/pipelines/azure-pipelines-external-release.yml

```diff
@@ -36,25 +42,17 @@ jobs:
     displayName: Use .NET Core sdk 8.0.x
     inputs:
       version: 8.0.x
-  - task: NuGetToolInstaller@1
-    displayName: Nuget Tool Installer
-    inputs:
-      versionspec: '*'
-      checkLatest: true
-  - task: NuGetAuthenticate@1
-    displayName: NuGet Authenticate
-  - task: NuGetAuthenticate@1
-    displayName: 'NuGet Authenticate'
 
   - task: DotNetCoreCLI@2
     displayName: dotnet build
     enabled: True
     inputs:
-      projects: '**/Garnet.*.csproj'
+      projects: '**/Garnet*.csproj'
       arguments: -c Release
 
   - task: PowerShell@2
     displayName: 'Publish the GarnetServer binaries'
+    enabled: True
```

Comment:
why is this line addition necessary?

Reply:
There are enabled: lines at each task which allows for easier development where you can disable all the tasks you want why developing pipelines.  I can remove them all.

Reply:
either keep all or remove all. it appeared from diff that only this section had it added, so.

Reply:
Removed all